From 62fbe9c3539945df6204501b7e3ae0d51db075a3 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Thu, 24 Nov 2016 22:30:40 +0100
Subject: [PATCH] Create a main include file QExtSerialPort

This main include file will be installed in
<QtExtSerialPort/QExtSerialPort> so that Qt applications can use this
library by including header files in a Qt-like style.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
[Arnout: add it to qextserialport.pri so it is installed]
Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
---
 src/QExtSerialPort     | 2 ++
 src/qextserialport.pri | 3 ++-
 2 files changed, 4 insertions(+), 1 deletion(-)
 create mode 100644 src/QExtSerialPort

diff --git a/src/QExtSerialPort b/src/QExtSerialPort
new file mode 100644
index 0000000..a48aeea
--- /dev/null
+++ b/src/QExtSerialPort
@@ -0,0 +1,2 @@
+#include "qextserialport.h"
+#include "qextserialenumerator.h"
diff --git a/src/qextserialport.pri b/src/qextserialport.pri
index 461d56f..f8a678d 100644
--- a/src/qextserialport.pri
+++ b/src/qextserialport.pri
@@ -3,7 +3,8 @@
 
 PUBLIC_HEADERS         += $$PWD/qextserialport.h \
                           $$PWD/qextserialenumerator.h \
-                          $$PWD/qextserialport_global.h
+                          $$PWD/qextserialport_global.h \
+			  $$PWD/QExtSerialPort
 
 HEADERS                += $$PUBLIC_HEADERS \
                           $$PWD/qextserialport_p.h \
